app/views/automated_tests/_test_script_result.html.erb

```diff
@@ -7,9 +7,9 @@
       <%= l(test_script_result.created_at, format: :long) %>
     </span>
   </h3>
-
-  <% tests = test_script_output['testrun']['test_script']['test'] %>
+  <% tests_scripts_outputs = test_script_output['testrun']['test_script'] %>
```

Comment:
This change suggests that there are multiple outputs and/or test scripts here. I don't think that this is the proper use of this view right now. I now get an error on line 26 which didn't occur before.

Please revert this change. We will deal with multiple test scripts later.